internalshared/configutil/env_var_util.go

```diff
@@ -81,4 +81,17 @@ var (
 		transit.EnvTransitWrapperDisableRenewal:   "disable_renewal",
 		transit.EnvVaultTransitSealDisableRenewal: "disable_renewal",
 	}
+
+	TransitPrioritizeConfigValues = []string{
```

Comment:
Suggestion: add an explanatory comment to `TransitPrioritizeConfigValues` and `TransitTlsConfigVars`.